2025-05-10 14:56:31,870 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-05-10 14:56:31,871 INFO supervisord started with pid 1
2025-05-10 14:56:32,876 INFO spawned: 'nginx' with pid 7
2025-05-10 14:56:32,883 INFO spawned: 'php-fpm' with pid 8
2025-05-10 14:56:32,914 CRIT uncaptured python exception, closing channel <POutputDispatcher at 133053325446288 for <Subprocess at 133053343861904 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 14:56:33,920 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 14:56:33,920 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 14:56:40,928 WARN received SIGTERM indicating exit request
2025-05-10 14:56:40,929 INFO waiting for nginx, php-fpm to die
2025-05-10 14:56:40,930 WARN stopped: php-fpm (terminated by SIGTERM)
2025-05-10 14:56:40,933 INFO stopped: nginx (exit status 0)
2025-05-10 15:02:30,999 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-05-10 15:02:31,000 INFO supervisord started with pid 1
2025-05-10 15:02:32,005 INFO spawned: 'nginx' with pid 7
2025-05-10 15:02:32,009 INFO spawned: 'php-fpm' with pid 8
2025-05-10 15:02:32,036 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:32,036 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:32,037 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:32,037 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:32,037 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state STARTING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:33,049 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:02:33,049 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:02:56,560 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:02:56,560 CRIT uncaptured python exception, closing channel <POutputDispatcher at 131768610594320 for <Subprocess at 131768596677648 with name php-fpm in state RUNNING> (stderr)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:03:03,567 WARN received SIGTERM indicating exit request
2025-05-10 15:03:03,568 INFO waiting for nginx, php-fpm to die
2025-05-10 15:03:03,570 WARN stopped: php-fpm (terminated by SIGTERM)
2025-05-10 15:03:03,570 INFO reaped unknown pid 9 (terminated by SIGTERM)
2025-05-10 15:03:03,570 INFO reaped unknown pid 10 (terminated by SIGTERM)
2025-05-10 15:03:03,570 INFO reaped unknown pid 11 (terminated by SIGTERM)
2025-05-10 15:03:03,570 INFO reaped unknown pid 12 (terminated by SIGTERM)
2025-05-10 15:03:03,573 INFO stopped: nginx (exit status 0)
2025-05-10 15:06:25,198 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-05-10 15:06:25,200 INFO supervisord started with pid 1
2025-05-10 15:06:26,205 INFO spawned: 'php-cgi' with pid 7
2025-05-10 15:06:26,211 INFO spawned: 'nginx' with pid 8
2025-05-10 15:06:27,266 INFO success: php-cgi entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:06:27,266 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:10:48,538 WARN received SIGTERM indicating exit request
2025-05-10 15:10:48,542 INFO waiting for php-cgi, nginx to die
2025-05-10 15:10:48,545 INFO stopped: nginx (exit status 0)
2025-05-10 15:10:48,550 INFO stopped: php-cgi (exit status 0)
2025-05-10 15:11:32,114 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-05-10 15:11:32,117 INFO supervisord started with pid 1
2025-05-10 15:11:33,122 INFO spawned: 'php-cgi' with pid 7
2025-05-10 15:11:33,130 INFO spawned: 'nginx' with pid 8
2025-05-10 15:11:34,134 INFO success: php-cgi entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:11:34,134 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:11:38,858 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136246535620688 for <Subprocess at 136246538163472 with name nginx in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:11:38,924 CRIT uncaptured python exception, closing channel <POutputDispatcher at 136246535620688 for <Subprocess at 136246538163472 with name nginx in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2025-05-10 15:14:52,130 WARN received SIGTERM indicating exit request
2025-05-10 15:14:52,131 INFO waiting for php-cgi, nginx to die
2025-05-10 15:14:52,135 INFO stopped: nginx (exit status 0)
2025-05-10 15:14:52,138 INFO stopped: php-cgi (exit status 0)
2025-05-10 15:15:35,703 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-05-10 15:15:35,706 INFO supervisord started with pid 1
2025-05-10 15:15:36,712 INFO spawned: 'php-cgi' with pid 7
2025-05-10 15:15:36,718 INFO spawned: 'nginx' with pid 8
2025-05-10 15:15:37,722 INFO success: php-cgi entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-05-10 15:15:37,722 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
